### PR TITLE
Add aliases to mdx frontmatter for Algolia testing

### PR DESCRIPTION
**Pull request by pfd**
## Description of the change
This PR adds some aliases to a few mdx front matter so we can test to make sure that Algolia is picking them up.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> N/A

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/index.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "Android"
 description: >-
   Detailed technical documentation on RudderStack’s Android SDK using Android
   Studio to send events from your Android device to various destinations.
-keywords: ["android", "android sdk", "android source"]
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/index.mdx:7]: this was suggested to us by Red Crackle. I believe the implementation has changed now? cc @pfd

**Review by ameypv-rudder — APPROVED** [review]
LGTM

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/index.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: "Android"
...
+aliases: ["android", "android sdk", "android source"]
```
Comment (ameypv-rudder) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/index.mdx:7]: I believe the casing won't matter for these keywords?

Comment (ishachopra0401) [reply stream-sources/rudderstack-sdk-integration-guides/rudderstack-android-sdk/index.mdx:7]: IMO, it should not. The current search results on our doc site are also case-independent.

**Pull request merged**